site2/docs/administration-zk-bk.md

```diff
@@ -183,9 +183,15 @@ BookKeeper bookies can be configured using the [`conf/bookkeeper.conf`](referenc
 
 You can start up a bookie in two ways: in the foreground or as a background daemon.
 
-To start up a bookie in the foreground, use the [`bookeeper`](reference-cli-tools.md#bookkeeper)
+To start up a bookie in the foreground, use the [`bookeeper`](reference-cli-tools.md#bookkeeper) CLI tool:
 
-```shell
+```bash
+$ bin/bookkeeper bookie
```

Comment:
I think you can use `bin/pulsar bookie`, no?

Reply:
yes, it is another choose, however `bin/pulsar` uses `pulsar_env.sh` by default, when someone trying to pass extra opts in `bkenv.sh` this shell cann't work, so here just keeps the same as `deploy-bare-metal` to guide users to start a bookie in foreground.